collect_monitoring.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Why do we need to target sandbox if we are running a docker image for FTP locally?